msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/IBroker.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.util.Set;
+
+/**
+ * Used to define the basic set of methods that all Brokers must implement
+ *
+ * All methods are so they can be referenced by MSAL Java without an implementation, and by default simply throw an
+ * exception saying that a broker implementation is missing
+ */
+public interface IBroker {
```

Comment:
You are missing a `RemoveAccount` functionality.

Reply:
Added.